src/enzo/CallProblemSpecificRoutines.C

```diff
@@ -42,7 +42,7 @@ int CallProblemSpecificRoutines(TopGridData * MetaData, HierarchyEntry *ThisGrid
     ThisGrid->GridData->SphericalInfallGetProfile(level, 1);
   if (ProblemType == 30)
     ThisGrid->GridData->AnalyzeTrackPeaks(level, 0);
-  if (ProblemType == 27){
+  if (ProblemType == 27 && debug1){
```

Comment:
I seem to recall that `debug1` results in a lot of output. Would `debug` be better?

Reply:
I agree that `debug` would be better.

Reply:
I agree that `debug` would be better.